.github/workflows/auto_request_review.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Request Review
+
+on:
+  pull_request:
+    types: [ opened, reopened ]
```

Comment:
Will this work when we move a PR from "Draft" to "Ready for review" or something?

Reply:
Yes, will test that scenario as well.